lib/Scheduler/sched_region.cpp

```diff
@@ -386,8 +438,12 @@ FUNC_RESULT SchedRegion::FindOptimalSchedule(
     if (AcoTime > 0)
       Logger::Info("ACO_Time %d", AcoTime);
 
+    InstCount acoExecCost;
+
     AcoScheduleLength_ = AcoSchedule->GetCrntLngth();
+    CmputNormCost_(AcoSchedule, CCM_DYNMC, acoExecCost, true);
```

Comment:
This is redundant. ACO already called this so AcoSchedule should already have all costs related information.